contributing.md

```diff
@@ -49,27 +49,80 @@ Design proposed and theme changes are present at: [Zeplin](https://scene.zeplin.
 
 # How Can I Contribute?
 
-### First Timers
+## Contributing Code
+
+### Step 1: Set up a working copy on your computer
+
+Firstly you need to set up a local fork of our project, so press the "fork" button in Github. This will create a copy of our project in your Github account.
+
+Then you can clone the project to your local machine using this command：
```

Comment:
"Then you can clone the project to your local machine by entering the following command into a terminal window:"